lib/rubocop/cop/money/missing_currency.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Money
+      class MissingCurrency < Cop
+        # `Money.new()` without a currency argument cannot guarantee correctness:
+        # - no error raised for cross-currency computation (e.g. 5 CAD + 5 USD)
+        # - #subunits returns wrong values for 0 and 3 decimals currencies
+        #
+        # @example
+        #   # bad
+        #   Money.new(123.45)
+        #   "1,234.50".to_money
+        #
+        #   # good
+        #   Money.new(123.45, 'CAD')
+        #   "1,234.50".to_money('CAD')
+        #
+
+        def_node_matcher :money_new_without_currency?, <<~PATTERN
+          (send (const nil? :Money) :new _)
+        PATTERN
+
+        def_node_matcher :to_money_without_currency?, <<~PATTERN
+          (send _ :to_money)
+        PATTERN
+
+        def_node_matcher :to_money_block?, <<~PATTERN
+          (send _ _ (block_pass (sym :to_money)))
+        PATTERN
+
+        def on_send(node)
+          if money_new_without_currency?(node)
+            add_offense(node, message: 'Money is missing currency argument')
+          end
+
+          if to_money_block?(node) || to_money_without_currency?(node)
+            add_offense(node, message: 'to_money is missing currency argument')
+          end
+        end
+
+        def autocorrect(node)
```

Comment:
do we need autocorrect? Seems a bit dangerous

Reply:
See the readme ;)